src/studio/src/designer/backend/Views/ServiceDevelopment/index.cshtml

```diff
@@ -27,8 +27,14 @@
 <link rel="stylesheet" href="~/designer/css/react/app-development.css">
 
 <div id="root" class="media-body flex-column d-flex"></div>
-<script src="~/designer/js/react/app-development.js" asp-append-version="true" type="text/javascript"></script>
+@* <script src="~/designer/js/react/app-development.js" asp-append-version="true" type="text/javascript"></script>
 <script src="~/designer/js/react/1.app-development.js" asp-append-version="true" type="text/javascript"></script>
 <script src="~/designer/js/react/2.app-development.js" asp-append-version="true" type="text/javascript"></script>
 <script src="~/designer/js/react/3.app-development.js" asp-append-version="true" type="text/javascript"></script>
-<script src="~/designer/js/react/4.app-development.js" asp-append-version="true" type="text/javascript"></script>
+<script src="~/designer/js/react/4.app-development.js" asp-append-version="true" type="text/javascript"></script> *@
+
+<script src="http://localhost:8080/app-development.js" asp-append-version="true" type="text/javascript"></script>
+<script src="http://localhost:8080/1.app-development.js" asp-append-version="true" type="text/javascript"></script>
+<script src="http://localhost:8080/2.app-development.js" asp-append-version="true" type="text/javascript"></script>
+<script src="http://localhost:8080/3.app-development.js" asp-append-version="true" type="text/javascript"></script>
+<script src="http://localhost:8080/4.app-development.js" asp-append-version="true" type="text/javascript"></script>
```

Comment:
This needs to be reverted, this is something that is only valid for local development together with webpack-dev-server. We should look into solving this in a better way to avoid having to make these changes, I created a task to look into this https://github.com/Altinn/altinn-studio/issues/8959.

Reply:
done